Makefile

```diff
@@ -1,4 +1,7 @@
 test:
 	bin/mocha specs
 
-.PHONY: test
+eslint:
+	node_modules/.bin/eslint .
+
+.PHONY: test eslint
```

Comment:
`eslint` is not necessary in PHONY, because it is not a special command for make utility